kalite/coachreports/static/js/coachreports/student_progress.js

```diff
@@ -101,10 +101,19 @@ var StudentProgressContainerView = Backbone.View.extend({
 
     initialize: function() {
         this.listenTo(this.collection, 'add', this.add_one);
+        
+        var self=this;
 
         this.render();
 
-        this.collection.fetch();
+        this.collection.fetch({
+                success: function() {
+                     if(self.collection.length==0)            //if the student visits the my progress page before attempting any quizes/videos
+                          {
+			      self.$el.html("<h>You have not started your learning journey yet.</h><br>Click on the LEARN button above to 						get started");
```

Comment:
This should be a django message rather than a plain html text we stick into the page.